### ISSUE TITLE
Tests broken with composer issue

### ISSUE THREAD

**Issue by rfay** [drud/ddev-selenium-standalone-chrome#10]
I see the tests are broken with a composer issue, https://github.com/drud/ddev-selenium-standalone-chrome/actions/runs/3827532283/jobs/6512197025 :

```
#     - laminas/laminas-diactoros 2.14.0 requires php ^7.3 || ~8.0.0 || ~8.1.0 -> your php version (8.2.0) does not satisfy that requirement.
#     - drupal/core-recommended 9.5.0 requires laminas/laminas-diactoros ~2.14.0 -> satisfiable by laminas/laminas-diactoros[2.14.0].
#     - drupal/core-recommended is locked to version 9.5.0 and an update of this package was not requested.
#
# ::error ::Your requirements could not be resolved to an installable set of packages.%0A%0A  Problem 1%0A    - laminas/laminas-diactoros 2.14.0 requires php ^7.3 || ~8.0.0 || ~8.1.0 -> your php version (8.2.0) does not satisfy that requirement.%0A    - drupal/core-recommended 9.5.0 requires laminas/laminas-diactoros ~2.14.0 -> satisfiable by laminas/laminas-diactoros[2.14.0].%0A    - drupal/core-recommended is locked to version 9.5.0 and an update of this package was not requested.%0A
```

**Comment by weitzman** [comment]
Looks like laminas now allows php 8.2 but drupal has not released a core-dev that allows it. This will resolve itself once Drupal does that.

**Comment by weitzman** [comment]
Oh, I need to run the composer commands *in* the container. Sound good?

**Comment by weitzman** [comment]
Fix in progress at #11

**Comment by rfay** [comment]
Thanks!
